slms/issue 8

josuedani opened the pull request:
Primera versión de micro servicio de cursos

Commit:
ADD Course ms

First version by pull request

Commit:
ADD pagination

Add pagination by params

garfenter commented on course-ms/routes.js:
```diff
@@ -0,0 +1,65 @@
+
+module.exports = [
+    {
+        path: '/{limit}/{offset}',
```
Limit y offset deberían de ser definidos como parámetros de query ya que podrían ser opcionales.

garfenter commented on course-ms/handlers.js:
```diff
@@ -0,0 +1,95 @@
+    });
+}
+
+// Find Couse by dbid
```
Typo en comentario: "Couse"

garfenter commented on course-ms/handlers.js:
```diff
@@ -0,0 +1,95 @@
+    institution: Joi.number().required(),
+    code: Joi.string().required(),
+    name: Joi.string().required(),
+    description: Joi.string().required(),
```
Creo que descripción podría ser un campo opcional

Commit:
Correcciones Pull Request

Descripción cambia opcional.
Cambio de params a query en pagination
Cambio en typo Couse to Course

Commit:
Actualiza Postman

Cambia de params to query.

Commit:
Add entity type_tag

Change Basepath
Add tags.handlers
Add course.handlers

Commit:
Entity Tag

Change name to tags to tagsTypes
Se crean rutas de entidad tags
se actualiza archivo de pruebas.
